src/components/Tool/Appeal/Modals/EditAppealHeaderInfoModal/EditAppealHeaderInfoModal.test.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SnackbarProvider } from 'notistack';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { AppealFieldsFragment } from 'pages/accountLists/[accountListId]/tools/GetAppeals.generated';
+import { AppealsWrapper } from 'pages/accountLists/[accountListId]/tools/appeals/AppealsWrapper';
+import theme from 'src/theme';
+import { appealInfo } from '../../appealMockData';
+import { EditAppealHeaderInfoModal } from './EditAppealHeaderInfoModal';
+
+const accountListId = 'abc';
+const router = {
+  query: { accountListId },
+  isReady: true,
+};
+const handleClose = jest.fn();
+const mutationSpy = jest.fn();
+
+const Components = ({
+  appeal = appealInfo,
+}: {
+  appeal?: AppealFieldsFragment;
+}) => (
+  <SnackbarProvider>
+    <DndProvider backend={HTML5Backend}>
+      <ThemeProvider theme={theme}>
+        <TestRouter router={router}>
+          <GqlMockedProvider onCall={mutationSpy}>
+            <AppealsWrapper>
+              <EditAppealHeaderInfoModal
+                appealInfo={appeal}
+                handleClose={handleClose}
+              />
+            </AppealsWrapper>
+          </GqlMockedProvider>
+        </TestRouter>
+      </ThemeProvider>
+    </DndProvider>
+  </SnackbarProvider>
+);
+
+describe('EditAppealHeaderInfoModal', () => {
+  it('should show errors', () => {
+    const { getByRole, getByTestId } = render(
+      <Components
+        appeal={{
+          ...appealInfo,
+          name: '',
+          amount: '' as unknown as AppealFieldsFragment['amount'],
```

Comment:
Instead of passing in an impossible value, I think a better test would be to `userEvent.clear()` the amount box. Then you can assert that there is an amount error message. What do you think about that?